src/guides/v2.3/install-gde/tutorials/change-docroot-to-pub.md

```diff
@@ -113,7 +113,19 @@ Replace `192.168.33.10` with your server's hostname.
    UPDATE core_config_data SET value='http://192.168.33.10' WHERE path='web/unsecure/base_url';
    ```
 
-## 3. Switch modes
+## 3. Update the env.php file
```

Comment:
Please add a blank line between the headers and the content. (L116 and L128. And the callout is unnecessary. It can be regular text.

Reply:
@dobooth,
Fixed. Thank you.